sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-integration.scheduler.e2e-spec.ts

```diff
@@ -98,7 +98,7 @@ describe(describeProcessorRootTest(QueueNames.IER12Integration), () => {
     // Intercept file timestamp.
     getFileNameAsCurrentTimestampMock = jest.spyOn(
       dateUtils,
-      "getFileNameAsCurrentTimestamp",
+      "getFileNameAsCustomizedCurrentTimestamp",
```

Comment:
Can you please elaborate on the motivation for renaming this?

Reply:
The jest.spyOn checks for the created filename by using the dateUtils particular method that we have mentioned. This is used to fetch the timestampResult in the testcases for assert, if this is not properly parsed in the jest spyOn the timestampResult was retrieved as undefined and the test cases were failing, this is were i told you created the test cases on jest.spyOn efficiently.
<img width="690" alt="image" src="https://github.com/bcgov/SIMS/assets/62901416/a88a8b17-66c4-4b99-ba44-80eb57cb9e46">

Reply:
@guru-aot I was just asking why we were using another method here. I got the reason from the date formating change 😉